mne/io/eyelink/eyelink.py

```diff
@@ -284,6 +287,45 @@ def _find_overlaps(df, max_time=0.05):
     return ovrlp.drop(columns=tmp_cols).reset_index(drop=True)
 
 
+@fill_doc
+def read_eyelink_calibration(
+    filename, screen_size=None, screen_distance=None, screen_resolution=None
+):
+    """Return info on calibrations collected in an eyelink file.
+
+    Parameters
+    ----------
+    filename : str
+        Path to the eyelink file (.asc).
+    screen_size : tuple
```

Comment:
```suggestion
    screen_size : tuple of shape (2,)
```

For sequence, it's nice to specify the expected shape.